contracts/EntryPoint.sol

```diff
@@ -83,6 +83,27 @@ contract EntryPoint is StakeManager {
         require(success);
     }
 
+    /**
+     * execute a user op
+     * @param opIndex into into the opInfo array
+     * @param userOp the userOp to execute
+     * @param opInfo the opInfo filled by validatePrepayment for this userOp.
+     * @return collected the total amount this userOp paid.
+     */
+    function _executeUserOp(uint256 opIndex, UserOperation calldata userOp, UserOpInfo memory opInfo) private returns (uint256 collected) {
+        uint256 preGas = gasleft();
+        uint256 contextOffset = opInfo.contextOffset;
+        bytes memory context;
+        assembly {context := contextOffset}
+
+        try this.innerHandleOp(userOp, opInfo, context) returns (uint256 _actualGasCost) {
+            collected = _actualGasCost;
+        } catch {
+            uint256 actualGas = preGas - gasleft() + opInfo.preOpGas;
```

Comment:
I understand that `preOpGas` is 'gas used before execution'?

Let me just point out that we have in this code:

- `verificationGas` which is a limit on gas for verification
- `preVerificationGas` which is an exact value for gas overhead to be added
- `preOpGas` which is a partial gas used value

I *strongly suggest* we spend some more letters in the names and call these `XxxLimit`, `XxxUsed`, `XxxOverhead` etc.

Reply:
ok. but its a shotgun renaming. I'll add it in a separate PR.